integration-tests/contract-execute-on-chain-view.spec.ts

```diff
@@ -75,10 +75,10 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
         await contract.contractViews.test_failwith(3).executeView({ viewCaller: contract.address });
       } catch (error: any) {
         const protocol = (await Tezos.rpc.getProtocols()).protocol
-        if(protocol === Protocols.PtKathman) {
-          expect(error).toBeInstanceOf(ViewSimulationError)
-        } else {
+        if(protocol === Protocols.PtJakart2) {
```

Comment:
Why is it changed from K to J?

Reply:
I reversed the if/else statement so Kathmandunet, mondaynet, and all incoming protocols use the else case without us having to maintain anything.